### DISCUSSION TITLE
Feature suggestion : Mobile App

### DISCUSSION THREAD

**Discussion by Dryhb** [LAION-AI/Open-Assistant#3441] [Ideas]
Considering https://github.com/LAION-AI/Open-Assistant/issues/974 (sorting assistant answer is tricky) and https://github.com/LAION-AI/Open-Assistant/issues/1202 (scrolling while sorting assistant answer changes order in the background) and https://github.com/LAION-AI/Open-Assistant/issues/1343 (the fact that the main part of the page can be moved horizontally) with the added "general" website things such as the top bar with URL appearing when scrolling up and depending on the app used a bottom bar can also appear (for example when you are in a tab group on chrome & others).  A mobile app would give a better user experience on mobile and could use the system's language to set the default language selected as this one (still able to change it later of course). It can be a port of the website or a little rework more suited for phone, if you can take a look at google crowdsource app I think the layout is good.

**Comment by mm12** [comment]
Perhaps rather than a full app, it could be a Progressive Web App (PWA) -- basically, it has a special layout, and can be added as an icon on the home screen, but in the back is just running the website with a different UI.

[a comment by an automated account is not shown]
